src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -18,3 +18,9 @@ It remains possible to configure your own logging behavior by setting the `exclu
 In addition, the `SimpleMessageListenerContainer` consumer restart after such an exception is now logged at DEBUG level by default (previously INFO).
 A new method `logRestart()` has been added to the `ConditionalExceptionLogger` to allow this to be changed.
 See xref:amqp/receiving-messages/consumer-events.adoc[Consumer Events] and <<channel-close-logging>> for more information.
+
+=== Connections Enhancement
+
+Connection Factory supported backoff policy when creating connection channel.See xref:amqp/connections.adoc[Choosing a Connection Factory] and https://github.com/spring-projects/spring-amqp/pull/2556[2556] for more information.
```

Comment:
One sentence per line, please: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

No PR links from the docs, please.

The `[[x31-conn-backoff]]` `id` is missed on this section.